Resources/ConfigPresets/Corvax/elysium.toml

```
@@ -1,7 +1,7 @@
 ﻿[game]
 hostname = "🛰️ Corvax — Элизиум 🌑"
 desc = "Корвакс первый русскоязычный проект в Space Station 14.\nЗдесь помогут новичкам!\nНа этом сервере пониженные требования к RP."
-map_pool = "CorvaxMapPool"
+map_pool = "CorvaxDefaultMapPool"
 maxplayers = 180
 soft_max_players = 100
 map_rotation = false
```

в таком случае и тут включить
```suggestion
map_rotation = true
```